sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -208,15 +211,14 @@ def detect_language(  # type: ignore
                 **kwargs
             )
         except HttpResponseError as error:
-            process_http_response_error(error)
+            return process_http_response_error(error)
```

Comment:
since process_http_response_error raises itself, is it necessary to return it?

Reply:
This was to appease mypy and remove the type: ignore statement on the function signature line. It complains that there is no return statement in the method.